bedrock/settings/base.py

```diff
@@ -118,17 +118,17 @@ def path(*args):
 
 # Accepted locales
 PROD_LANGUAGES = ('ach', 'af', 'an', 'ar', 'as', 'ast', 'az', 'be', 'bg',
-                  'bn-BD', 'bn-IN', 'br', 'brx', 'bs', 'ca', 'cs', 'cy',
-                  'da', 'de', 'dsb', 'ee', 'el', 'en-GB', 'en-US', 'en-ZA',
-                  'eo', 'es-AR', 'es-CL', 'es-ES', 'es-MX', 'et', 'eu',
-                  'fa', 'ff', 'fi', 'fr', 'fy-NL', 'ga-IE', 'gd', 'gl',
-                  'gu-IN', 'ha', 'he', 'hi-IN', 'hr', 'hsb', 'hu',
+                  'bn-BD', 'bn-IN', 'br', 'brx', 'bs', 'ca', 'cak', 'cs',
+                  'cy', 'da', 'de', 'dsb', 'ee', 'el', 'en-GB', 'en-US',
+                  'en-ZA', 'eo', 'es-AR', 'es-CL', 'es-ES', 'es-MX', 'et',
+                  'eu', 'fa', 'ff', 'fi', 'fr', 'fy-NL', 'ga-IE', 'gd',
+                  'gl','gu-IN', 'ha', 'he', 'hi-IN', 'hr', 'hsb', 'hu',
```

Comment:
Looks like a missing trailing whitespace after `,` here is causing Travis to complain.